distribution/sdk/sample-mgmp-server/README.md

```diff
@@ -0,0 +1,126 @@
+          },
+          "request": {
+            "matches": "GetCapabilities"
+          },
+          "service": {
+            "matches": "CSW"
+          }
+        }
+    },
+    "response": {
+            "status": 200,
+            "headers" : {
+                "Content-Type": "text/xml"
+            },
+            "bodyFileName": "MgmpGetCapabilitiesResponse.xml"
+    }
+}
+```
+
+##Mock MGMP Server configuration
+
+The Mock MGMP Server configuration lives in the sample-mgmp-server pom.xml.  Things such as ports, etc. can be changed/added by modifying/adding arguments to the <arguments> section.  The server is currently set to run on port 9081.  See http://wiremock.org/running-standalone.html for supported command line options.
+
+```xml
+    <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>1.3.2</version>
+        <executions>
+            <execution>
+                <goals>
+                    <goal>java</goal>
+                </goals>
+            </execution>
+        </executions>
+        <configuration>
+           <includePluginDependencies>true</includePluginDependencies>
+           <includeProjectDependencies>false</includeProjectDependencies>
+           <executableDependency>
+               <groupId>com.github.tomakehurst</groupId>
+               <artifactId>wiremock</artifactId>
+           </executableDependency>
+           <mainClass>com.github.tomakehurst.wiremock.standalone.WireMockServerRunner
+           </mainClass>
+           <arguments>
+                <argument>--port</argument>
+                <argument>9081</argument>
+                <argument>--https-port</argument>
+                <argument>9443</argument>
+                <argument>--root-dir</argument>
+                <argument>src/main/resources/mock-mgmp</argument>
+                <argument>--verbose</argument>
+            </arguments>
+        </configuration>
+        <dependencies>
+            <dependency>
+                <groupId>com.github.tomakehurst</groupId>
+                <artifactId>wiremock</artifactId>
+                <classifier>standalone</classifier>
+                <version>1.53</version>
+            </dependency>
+        </dependencies>
+    </plugin>
+```
+
+##Using the Mock MGMP Server
+
+1. Navigate to the sample-mgmp-server directory.
+2. Run `mvn exec:java`
+3. Configure an MGMP Federated source using http://localhost:9081/csw and default settings
+
+Below is what one will see when the Mock MGMP Server is running.
+
+```
+$ mvn exec:java
+[INFO] Scanning for projects...
+[INFO]
+[INFO] ------------------------------------------------------------------------
+[INFO] Building Alliance :: Distribution :: SDK :: Sample MGMP Server 0.2-SNAPSHOT
+[INFO] ------------------------------------------------------------------------
+[INFO]
+[INFO] --- exec-maven-plugin:1.3.2:java (default-cli) @ sample-mgmp-server ---
```

Comment:
I'm a little worried about these specific version strings here and above in the pom snippet. Either they'll quickly fall out of date when the mock server is updated and when the next version of Alliance is cut, or we'll constantly be updating them here. It's not a huge deal but worth considering.